dashboard/src/pages/TreeDetails/Tabs/LogSheet.tsx

```diff
@@ -121,8 +126,29 @@ export const LogSheet = ({
     { logUrl: logUrl ?? '' },
     { enabled: !!logUrl },
   );
+
+  const keyboardMap: { [key: string]: (() => void) | undefined } = {
+    ArrowRight: navigationLogsActions?.nextItem,
+    ArrowLeft: navigationLogsActions?.previousItem,
+    ArrowDown: navigationLogsActions?.nextItem,
+    ArrowUp: navigationLogsActions?.previousItem,
+    h: navigationLogsActions?.previousItem,
+    j: navigationLogsActions?.nextItem,
+    k: navigationLogsActions?.previousItem,
+    l: navigationLogsActions?.nextItem,
+  };
```

Comment:
We can improve this by:
- declaring the map in the module scope
- create functions that will handle the goNext and goPrevious (also the de module scope)